crates/router/src/utils.rs

```diff
@@ -1344,3 +1344,7 @@ pub async fn trigger_refund_outgoing_webhook(
 ) -> RouterResult<()> {
     todo!()
 }
+
+pub fn is_webhook_not_found_error(err: &errors::ApiErrorResponse) -> bool {
+    matches!(err, errors::ApiErrorResponse::WebhookResourceNotFound)
```

Comment:
Please add all the other NotFound errors which can occur like PaymentNotFound, RefundNotFound etc.